apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -115,6 +115,8 @@ const (
 // mechanism
 type AuthPodIdentity struct {
 	Provider PodIdentityProvider `json:"provider"`
+	// +optional
+	IdentityID string `json:"identityID"`
```

Comment:
See doc PR but we typically use `Id` over `ID`
```suggestion
	IdentityId string `json:"identityId"`
```